tests/test_dle.py

```diff
@@ -616,6 +616,73 @@ def test_incs_no_atomic(self):
         assert 'collapse(1)' in str(op1)
         assert 'atomic' not in str(op1)
 
+    @pytest.mark.parametrize('exprs,exp_simd', [
+        # none (different distance)
+        (['Eq(y.symbolic_max, g[0, x], implicit_dims=(t, x))',
+         'Inc(h1[0, 0], 1, implicit_dims=(t, x, y))'],
+         1),
+        (['Eq(y.symbolic_max, g[0, x], implicit_dims=(t, x))',
+         'Eq(h1[0, 0], y, implicit_dims=(t, x, y))'],
+         1),
+        (['Eq(y.symbolic_max, g[0, x], implicit_dims=(t, x))',
+         'Eq(h1[0, y], y, implicit_dims=(t, x, y))'],
+         1),
+        (['Eq(y.symbolic_min, g[0, x], implicit_dims=(t, x))',
+         'Eq(h1[0, y], 3 - y, implicit_dims=(t, x, y))'],
+         1),
+        (['Eq(y.symbolic_min, g[0, x], implicit_dims=(t, x))',
+          'Eq(y.symbolic_max, g[0, x], implicit_dims=(t, x))',
+          'Eq(h1[0, y], y, implicit_dims=(t, x, y))'],
+         1),
+        (['Eq(y.symbolic_min, g[0, 0], implicit_dims=(t, x))',
+          'Eq(y.symbolic_max, g[0, 2], implicit_dims=(t, x))',
+          'Eq(h1[0, y], y, implicit_dims=(t, x, y))'],
+         0),
+        (['Eq(y.symbolic_min, g[0, x], implicit_dims=(t, x))',
+          'Eq(y.symbolic_max, g[0, 2], implicit_dims=(t, x))',
+          'Inc(h1[0, y], y, implicit_dims=(t, x, y))'],
+         0),
+        (['Eq(y.symbolic_min, g[0, x], implicit_dims=(t, x))',
+          'Eq(y.symbolic_max, g[0, 2], implicit_dims=(t, x))',
+          'Inc(h1[0, x], y, implicit_dims=(t, x, y))'],
+         0),
+        (['Eq(y.symbolic_min, g[0, 0], implicit_dims=(t, x))',
+          'Inc(h1[0, y], x, implicit_dims=(t, x, y))'],
+         1),
+        (['Eq(y.symbolic_min, g[0, 2], implicit_dims=(t, x))',
+          'Inc(h1[0, x], y.symbolic_min, implicit_dims=(t, x))'],
+         0),
+        (['Eq(y.symbolic_min, g[0, 2], implicit_dims=(t, x))',
+          'Inc(h1[0, x], y.symbolic_min, implicit_dims=(t, x, y))'],
+         0),
+        (['Eq(y.symbolic_min, g[0, 2], implicit_dims=(t, x))',
+          'Eq(h1[0, x], y.symbolic_min, implicit_dims=(t, x))'],
+         0),
+        (['Eq(y.symbolic_min, g[0, x], implicit_dims=(t, x))',
+          'Eq(y.symbolic_max, g[0, x]-1, implicit_dims=(t, x))',
+          'Eq(h1[0, y], y, implicit_dims=(t, x, y))'],
+         0)
+    ])
+    def test_omp_pragmas_edge_cases(self, exprs, exp_simd):
```

Comment:
nitpicking: "omp" redundant as you already are in a class testing openmp stuff. "pragmas" also unnecessary probably